static/js/tab_bar.js

```diff
@@ -96,7 +96,6 @@ function make_tab_data() {
         // Third breadcrumb item for stream-subject naarrows
         if (filter.has_operator("stream") &&
             filter.has_operator("topic")) {
-            stream = filter.operands("stream")[0];
```

Comment:
It looks like `stream` was intended to be used in some way, otherwise the check `filter.has_operator("stream")` above would be pointless.